io/post_regional.F90

```diff
@@ -2905,7 +2922,7 @@ subroutine set_postvars_regional(wrt_int_state,mpicomp,setvar_atmfile,   &
 
 !htop
       do j=jsta,jend
-        do i=1,im
+        do i=ista,iend
```

Comment:
Why are this do loop bounds changed to `ista,iend`, while all of the above loops bound are `iista,iiend`? Are these bounds the same?

Reply:
I understand that. My question is why is this loop changed to `do i=ista,iend` and not `do i=iista,iiend`?

Reply:
`iista` and `iiend` are local variables defined here:
https://github.com/NOAA-EMC/fv3atm/blob/90bd8e8df293677a12e0cef42e104f587a0538f9/io/post_regional.F90#L529
and are initialized here:
https://github.com/NOAA-EMC/fv3atm/blob/90bd8e8df293677a12e0cef42e104f587a0538f9/io/post_regional.F90#L902

Which means they are the same during the execution of this routine. That means there's no need to use `iista` and `iiend`, `ista` and `iend` should be used. Why did you introduce `iista` and `iiend`? Please remove them and change all do loops to just use `ista` and `iend`. Code will be much more consistent.

Reply:
`iista` and `iiend` are local variables defined here:
https://github.com/NOAA-EMC/fv3atm/blob/90bd8e8df293677a12e0cef42e104f587a0538f9/io/post_regional.F90#L529
and are initialized here:
https://github.com/NOAA-EMC/fv3atm/blob/90bd8e8df293677a12e0cef42e104f587a0538f9/io/post_regional.F90#L902

Which means they are the same during the execution of this routine. That means there's no need to use `iista` and `iiend`, `ista` and `iend` should be used. Why did you introduce `iista` and `iiend`? Please remove them and change all do loops to just use `ista` and `iend`. Code will be much more consistent.

Reply:
@junwang-noaa Do you understand why we need two sets of I-index bounds in this routine `ista`,`iend` and also `iista`,`iiend` with the same values?